### DISCUSSION TITLE
Make Serverpod more like Supabase

### DISCUSSION THREAD

**Discussion by tananga** [serverpod/serverpod#2908] [Ideas]
Hello Serverpod team. I wonder if its gonna be hard to convert your architecture from current version to one which is more like in Supabase case. I was following tutorials prepared by Serverpod team but it was not as easy to use as in Supabase case. Also when I was generating "client" folder the build process was successful but for some reason I had a lot of errors in "client" folder and I had to clean cache, delete folder, get packages and generate it again.

**Comment by Isakdl** [comment]
Hello @Tananga.

Could you expand a little on what you mean? What specifically is it that you are looking for, what is easier with Supabase, that is missing in Serverpod?

You have to run `dart pub get` in the client library to install all required dependencies.
